lib/linguist/languages.yml

```diff
@@ -7409,6 +7409,18 @@ Windows Registry Entries:
   codemirror_mode: properties
   codemirror_mime_type: text/x-properties
   language_id: 969674868
+WebAssembly Interface Type:
+  type: programming
+  color: "#6250e7"
+  extensions:
+  - ".wit"
+  aliases:
+  - wit
+  ace_mode: text
+  tm_scope: source.wit
+  codemirror_mode: clike
+  codemirror_mime_type: text/x-csrc
```

Comment:
```suggestion
  codemirror_mode: webidl
  codemirror_mime_type: text/x-webidl
```

CodeMirror's [C-like mode](https://codemirror.net/5/mode/clike/) fails to highlight the following WIT fragment:

~~~
world my-world {
	import host: self.host
	import other-functionality: pkg.sibling-file
	import more-functionality: pkg.sibling-file.specific-interface
}

interface host {
	// ...
}
~~~

Its [WebIDL mode](https://codemirror.net/5/mode/webidl/index.html), OTOH, does a better—albeit still imperfect—job of highlighting WIT source. Having some highlighting is better than none at all.